### ISSUE TITLE
Tests in the "tests" folder don't run

### ISSUE THREAD

**Issue by kstenerud** [nguyenchr/atom-cargo-test-runner#12]
The tests in `src/lib.rs` run just fine, but if I open a file in my tests folder like `tests/test_something.rs` and try to run it, I just get "0 passed, 0 failed, 0 ignored, 0 measured".

**Comment by binary132** [comment]
Same.  When I run `cargo test` from the command line in the crate root, it runs the `tests/...` files just fine.

When I turned on the "Show Context Information" settings option and use this test runner, it shows:

```
Cargo binary:   /usr/local/bin/cargo
Root folder:    /Users/bodiesolomon/meta/dev/workspace/workshop/scratchpad/rust/env
Test file:      env_test.rs
PATH:           .:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:.:

... (the rest of the output is cut off and I can't scroll down.)
```

**Comment by binary132** [comment]
This Issue is a duplicate of #4, where I left the solution in a comment.

**Comment by nguyenchr** [comment]
this fix is now in the latest `v0.1.0` release, thanks for the help :)
